src/app/components/accounts/register/register.component.ts

```diff
@@ -1,23 +1,22 @@
-import {Component, OnInit} from '@angular/core';
-import {FormArray, FormBuilder, FormControl, FormGroup, Validators} from '@angular/forms';
+import {ChangeDetectionStrategy, Component, OnInit} from '@angular/core';
+import {AbstractControl, FormArray, FormBuilder, FormControl, FormGroup, Validators} from '@angular/forms';
 import {environment} from '@environments/environment';
 import {RegisterService} from '@app/_services/api/accounts/register.service';
 import {MessageService} from '@app/_services/message.service';
 import {confirmPasswordValidator} from '@app/_helpers/confirm-password.validator';
 import {MESSAGE_TYPES} from '@app/_models';
-import {AuthenticationService} from '@app/_services/api/authentication';
-import {first} from 'rxjs/operators';
 
 @Component({
     selector: 'app-register',
     templateUrl: './register.component.html',
-    styleUrls: ['./register.component.scss']
+    styleUrls: ['./register.component.scss'],
+    changeDetection: ChangeDetectionStrategy.OnPush
```

Comment:
what is this?

Reply:
It was required by the ESLint checker. It throws and error without this. This is what was recommended to include.

Reply:
I removed this from eslint. You can change it back.

Reply:
Okay cool